etc/adyen_payment.xml

```diff
@@ -77,5 +77,49 @@
             <label>Korean Local Card</label>
             <code_alt>korean_local_card</code_alt>
         </type>
+        <type id="AMEXAPPLEPAY" order="160">
+            <label>Apple Pay American express</label>
+            <code_alt>amex_applepay</code_alt>
+        </type>
+        <type id="DISCOVERAPPLEPAY" order="170">
+            <label>Apple Pay Discover</label>
+            <code_alt>discover_applepay</code_alt>
+        </type>
+        <type id="ELECTRONAPPLEPAY" order="180">
+            <label>Apple Pay Electron</label>
+            <code_alt>electron_applepay</code_alt>
+        </type>
+        <type id="ELOAPPLEPAY" order="190">
+            <label>Apple Pay Elo</label>
+            <code_alt>elo_applepay</code_alt>
+        </type>
+        <type id="ELODEBITAPPLEPAY" order="200">
+            <label>Apple Pay Elodebit</label>
+            <code_alt>elodebit_applepay</code_alt>
+        </type>
+        <type id="InteracAPPLEPAY" order="210">
+            <label>Apple Pay Interac</label>
+            <code_alt>interac_applepay</code_alt>
+        </type>
+        <type id="JCBAPPLEPAY" order="220">
+            <label>Apple Pay Jcb</label>
+            <code_alt>jcb_applepay</code_alt>
+        </type>
+        <type id="MAESTROAPPLEPAY" order="230">
+            <label>Apple Pay Maestro</label>
+            <code_alt>maestro_applepay</code_alt>
+        </type>
+        <type id="MCAPPLEPAY" order="240">
+            <label>Apple Pay Mastercard</label>
+            <code_alt>mc_applepay</code_alt>
+        </type>
+        <type id="VIAPPLEPAY" order="250">
+            <label>Apple Pay Visa</label>
+            <code_alt>visa_applepay</code_alt>
+        </type>
+        <type id="GIROCARDAPPLEPAY" order="260">
+            <label>Apple Pay Girocard</label>
+            <code_alt>girocard_applepay</code_alt>
+        </type>
```

Comment:
Would these options show in the config panel Card types? if so this might require an upgrade script so that they are enabled.

But if it does show then I don't think it would be great. As a merchant I would like to enable Visa once, for Cards, Apple Pay, Google Pay, etc. What do you think?

Reply:
The list for the config Credit Card Types is [here](https://github.com/Adyen/adyen-magento2/blob/develop/Model/Config/Source/CcType.php#L22) and we check if the allowed type is in the list. In that the case the cc type will be shown in the config.